experiment_scheduler/master/master.py

```diff
@@ -392,6 +395,14 @@ def execute_task(self, task_manager_address, task_id):
             task.commit()
         return response
 
+    @start_end_logger
+    def upload_file(self, request_iterator, context):
+        for request in request_iterator:
+            self.process_monitor.upload_file(request)
+        return MasterResponse(
+            experiment_id="0", response=MasterResponse.ResponseStatus.SUCCESS
+        )
```

Comment:
I think 'experiment_id' is unuseful.
So, I suggest that this method return new type of message, like 
```suggestion
        return UploadResponse(
            file_name = "(ex.uploaded file name)", response=MasterResponse.ResponseStatus.SUCCESS
        )
```